hnn_core/viz.py

```diff
@@ -6,9 +6,22 @@
 
 import numpy as np
 from itertools import cycle
+
+import matplotlib.colors as mc
```

Comment:
This import is causing the trouble. I'm not sure why it doesn't get triggered in the other examples. You should "re-run job with ssh" and then try to figure what's going on

Reply:
Actually, aren't matplotlib imports always nested? I think that's how we have done it in MNE in the past because of known issues. Probably it's not thread safe:

https://matplotlib.org/stable/users/faq/howto_faq.html#work-with-threads

Reply:
Bumping the matplotlib version up to >=3.5.3 seems solved the problem